src/conn/conn_api.c

```diff
@@ -2329,7 +2329,7 @@ __wt_verbose_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
       {"checkpoint", WT_VERB_CHECKPOINT}, {"checkpoint_cleanup", WT_VERB_CHECKPOINT_CLEANUP},
       {"checkpoint_progress", WT_VERB_CHECKPOINT_PROGRESS}, {"chunkcache", WT_VERB_CHUNKCACHE},
       {"compact", WT_VERB_COMPACT}, {"compact_progress", WT_VERB_COMPACT_PROGRESS},
-      {"configuration", WT_VERB_CONFIGURATION},
+      {"configuration", WT_VERB_CONFIGURATION}, {"default", WT_VERB_DEFAULT},
```

Comment:
As these logs capture sweep server operations, I'd prefer a name like WT_VERB_SWEEP or any identifier that specifically reflects sweep server activity.

Reply:
done